src/index.tsx

```diff
@@ -1,18 +1,11 @@
-import React from "react";
-import ReactDOM from "react-dom/client";
-import App from "./App";
-import reportWebVitals from "./reportWebVitals";
+import React from 'react';
+import { render } from 'react-dom';
+import App from './App';
+const root = document.getElementById('root');
 
-const root = ReactDOM.createRoot(
-  document.getElementById("root") as HTMLElement
-);
-root.render(
+render(
   <React.StrictMode>
     <App />
-  </React.StrictMode>
+  </React.StrictMode>,
```

Comment:
@mui/styles is not compatible with React.StrictMode

Reply:
Removed `@mui/styles` since it's deprecated. We can use `sx` property instead of `@mui/styles`, so we can keep the strict mode.